<launch>
    <arg name="republish_compressed_image" default="1"/>

    <node name="aruco_node" pkg="gripper_blaser_ros" type="aruco_node" output="screen" />
  
    <!-- Republish compressed image as raw image -->
    <group if="$(eval arg('republish_compressed_image') == 1)">
        <!-- <node name="visual_republish" pkg="image_transport" type="republish" args="compressed in:=/ximea_cam/image_raw raw out:=/fujun"/> -->
        <node name="visual_republish" pkg="image_transport" type="republish" args="compressed in:=/usb_cam/image_raw raw out:=/fujun"/>
    </group>

    <!-- rviz -->
    <!-- <node type="rviz" name="rviz" pkg="rviz" args="-d $(find gripper_blaser_ros)/cfg/rviz_config.rviz"/> -->
</launch>
